libraries/AP_Motors/AP_MotorsHeli_RSC.cpp

```diff
@@ -344,23 +359,35 @@ void AP_MotorsHeli_RSC::output(RotorControlState state)
 void AP_MotorsHeli_RSC::update_rotor_ramp(float rotor_ramp_input, float dt)
 {
     int8_t ramp_time;
+    int8_t bailout_time;
     // sanity check ramp time and enable bailout if set
-    if (_use_bailout_ramp || _ramp_time <= 0) {
+    if (_ramp_time <= 0) {
         ramp_time = 1;
     } else {
         ramp_time = _ramp_time;
     }
 
+    if (_rsc_bailout_time <= 0) {
+        bailout_time = 1;
+    } else {
+        bailout_time = _rsc_bailout_time;
+    }
+
     // ramp output upwards towards target
     if (_rotor_ramp_output < rotor_ramp_input) {
-        // allow control output to jump to estimated speed
-        if (_rotor_ramp_output < _rotor_runup_output) {
-            _rotor_ramp_output = _rotor_runup_output;
+        if (_use_bailout_ramp){
+            if(!_bailing_out){
+                gcs().send_text(MAV_SEVERITY_CRITICAL, "bailing_out");
```

Comment:
Again, not sure this is a  motors level thing.  We should put this stuff up in heli.  Also not MAV_SEVERITY_CRITICAL